src/components/PokeList/List/List.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * List of pokemons
+ */
+import Pokemon from '../pokemon/Pokemon';
+import './List.css';
+
+function List(props) {
+
+    return(
+        <ul className='pokeList'>
+            {props.pokemon.map(poke => 
```

Comment:
if you are mapping on a DOM element, you have to add a unique `key` property to the mapped element